code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -203,6 +203,14 @@
 				new /obj/effect/decal/cleanable/greenglow(T)
 			return
 
+/datum/reagent/platinum
+	name ="Platinum"
+	id = "platinum"
+	description = "Platinum is a naturally occuring silvery metalic element."
+	reagent_state = SOLID
+	color = "#E0E0E0"
+	taste_description = "a watch"
```

Comment:
Metallic watches are typically made of steel, I find this description offensive.